index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main">
+      <div id="title">Dr. Norman Borlaug</div>
```

Comment:
Please consider using heading tag for your title. Writing semantic HTML makes your code easier to understand, making the source code more readable for other developers.

Reply:
- [ ] Kindly provide context and an example for the student, don't just stop at `semantic reasons` and add some resources about the issue so the student can learn something new as well 👍🏼 🧑‍🎓